src/Core/Silk.NET.BuildTools/Common/Project.cs

```diff
@@ -11,6 +11,11 @@ namespace Silk.NET.BuildTools.Common
     /// </summary>
     public class Project
     {
+        /// <summary>
+        /// Gets or sets the class name of the project
+        /// </summary>
+        public string ClassName;
+
```

Comment:
Do we really need this? We have the class names in the `Classes`. An assumed 1:1 map of classes to projects is probably not an assumption we want.